addons/cloud_storage_azure/utils/cloud_storage_azure_utils.py

```diff
@@ -13,6 +13,7 @@
 from urllib.parse import quote
 from datetime import date
 from lxml import etree
+from odoo import _
```

Comment:
```suggestion

from odoo import _
```
While you are at it, you have the opportunity to make the imports closer to the guidelines :innocent: